app/styles/base/_typography.scss

```diff
@@ -10,7 +10,7 @@ body {
   background-position: center;
   background-repeat: no-repeat;
   background-attachment: fixed;
-  background-color: #2EE59D;
+  background-color: #ffffff;
```

Comment:
definitely less distracting, but it kinda makes the white buttons disappear leaving floating text for a split sec. What about something slightly gray, like #EBECEC?